openedx_events/learning/data.py

```diff
@@ -589,3 +589,23 @@ class BadgeData:
     uuid = attr.ib(type=str)
     user = attr.ib(type=UserData)
     template = attr.ib(type=BadgeTemplateData)
+
+
+@attr.s(frozen=True)
+class VerificationAttemptData:
+    """
+    Attributes defined for the Open edX IDV attempt data object.
+
+    Arguments:
+        attempt_id (int): the verification attempt id of the attempt being created or updated
```

Comment:
```suggestion
        attempt_id (int): the id of the verification attempt
```
Nit. I think this is a little clearer and doesn't tie the data class to a create or update action.